k8s/bigchaindb/bigchaindb-dep.yaml

```diff
@@ -12,7 +12,7 @@ spec:
       terminationGracePeriodSeconds: 10
       containers:
       - name: bigchaindb
-        image: bigchaindb/bigchaindb:1.3.0
+        image: bigchaindb/bigchaindb:unstable
```

Comment:
I wonder if we should change the `imagePullPolicy` on these "unstable" Docker images to `Always` --- because the content of the `bigchaindb/bigchaindb:unstable` image can change. Either that or we could adopt some naming convention for the "unstable" images such as `unstable1`, `unstable2`, etc.

Reply:
I think changing the `imagePullPolicy` is a better idea, I will take care of it.

Reply:
I think as long as, we keep it `unstable` i.e. till the official release, I will keep the policy as `Always`